VmAgent.Core/Interfaces/DockerContainerEngine.cs

```diff
@@ -614,7 +608,7 @@
                 if (_systemOperations.IsOSPlatform(OSPlatform.Windows))
                 {
                     var windowsStartCommand = new List<string>();
-                    
+
                     if (request.WindowsCrashDumpConfiguration?.IsEnabled == true)
```

Comment:
should we make it a no-op (return Task.CompletedTask) to avoid breaking anything in the future?